docs/tutorials/educators/intro.ipynb

```diff
@@ -307,15 +282,13 @@
    "source": [
     "We can unpack this a bit and see all of the components for the circuit.\n",
     "\n",
+    "The first thing we do is pick some qubits to use. There are many different types of qubits in Cirq, and you can define your own by inheriting from the `cirq.Qid` class. There's nothing inherently special or magical about these quantum id types such as `cirq.NamedQubit`. They simply identify what you wish to operate on, which is relevant when you are targeting a specific device. For example, if we were creating a circuit for the Bristlecone device and wanted to refer to the qubit in the left-most position, we would use `cirq.GridQubit(5, 0)`. (See the first diagram of the Bristlecone device we printed out.) For simplicity, in the previous cell we defined `cirq.NamedQubit`s which are simply qubits that can be identified by a name.\n",
     "\n",
-    "The first thing we do is pick some qubits to use. There are many different types of qubits in Cirq, and you can define your own by inheriting from the `cirq.Qid` class. There's nothing inherently special or magical about these quantum id types such as `cirq.NamedQubit`. They simply identify what you wish to operate on, which is relevant when you are targeting a specific device. For example, if we were creating a circuit for the Bristlecone device we would use `cirq.GridQubit(5, 0)` to refer to the qubit in the left most position of the device. To keep these simple for now, we'll start with abstract qubits simply identified by a name such as \"a\".\n",
-    "```\n",
-    "a = cirq.NamedQubit(\"a\")\n",
-    "```\n",
+    "Next, we encounter the object `cirq.H` which is a Hadamard gate with unitary\n",
     "\n",
-    "Next we encounter of the object `cirq.H`, which is a Hadamard gate.  `cirq.H` is an instance of the `cirq.HGate` class, which itself is a subclass of `Gate` (along with other classes). \n",
-    "$$H = {1 \\over \\sqrt{2}} \\left[ \\begin{array}[cc]  & 1 & 1  \\\\ 1 & -1 \\end{array}\\right]$$\n",
-    "We can use cirq to see this unitary matrix:"
+    "$$ H = {1 \\over \\sqrt{2}} \\left[ \\begin{array}[cc]  & 1 & 1  \\\\ 1 & -1 \\end{array}\\right] . $$\n",
```

Comment:
I _think_ you need newlines to render correctly (at least [on Github](https://github.com/quantumlib/Cirq/blob/4ee947775774de0fdf0ab02553f8c53b93c161a4/docs/tutorials/educators/intro.ipynb) it doesn't render well)
```suggestion
    "$$
     H = {1 \\over \\sqrt{2}} \\left[ \\begin{array}[cc]  & 1 & 1  \\\\ 1 & -1 \\end{array}\\right]
     $$\n",
```

Reply:
I still see a bunch of `$$` misrenderings on Github but maybe that's github's fault?

Reply:
It looks good on Jupyter notebook. Let's see how it behaves on devsite

Reply:
May the tex-in-jupyter-notebook gods be in our favor 🤞